modulemd/v2/include/modulemd-2.0/modulemd-component-rpm.h

```diff
@@ -192,4 +192,18 @@ modulemd_component_rpm_set_repository (ModulemdComponentRpm *self,
 const gchar *
 modulemd_component_rpm_get_repository (ModulemdComponentRpm *self);
 
+
+/**
+ * modulemd_component_rpm_equals:
```

Comment:
Don't make this a public function. It shouldn't be called directly. We expect consumers to use `modulemd_component_equals()` instead. This should be a static function defined only in `modulemd-component-rpm.c`

Reply:
Fixed